FEAT(server): Remove broadcast of new welcometext when changing via Ice

KetchupBomb opened the pull request:
When changing the welcome text via Ice, Murmur would broadcast the new text to all connected clients. It is a matter of opinion if this is proper behavior or not, but it seems too proactive of an action. Let the clients see the new welcometext when they next connect, or let the Murmur administrator explicitly perfor a sendMessage() to all connected clients after changing the welcometext.

Closes: #4788


### Checks

- [x] My commits follow the [commit guidelines](https://github.com/mumble-voip/mumble/blob/master/COMMIT_GUIDELINES.md)

Krzmbrzl commented on src/murmur/Server.cpp:
```diff
@@ -598,7 +598,6 @@ void Server::setLiveConf(const QString &key, const QString &value) {
 			qsWelcomeText = text;
 			MumbleProto::ServerConfig mpsc;
 			mpsc.set_welcome_text(u8(qsWelcomeText));
```
```suggestion
```

These can be removed as well. If we don't send the message, then there is no point in constructing it in the first place :)

KetchupBomb replied:
Should this deletion also include line 598? (`qsWelcomeText = text;`)

Krzmbrzl replied:
Exactly :)

Krzmbrzl reviewed (changes requested):
You commit also is not a refactoring as it does change the current behavior. I think this should probably be labeled as `FEAT(server)` instead.

If you want to attribute me in the change, please also specify my email from
https://github.com/mumble-voip/mumble/blob/384196a99f3446cff9984c873882a3eb37af0013/.mailmap#L121
so that GitHub will properly pick up on that :)

Krzmbrzl commented:
And on another note: It seems like you have used a different name/email as you have set up on GitHub. Otherwise the commit would show your profile picture next to the commit.
If you care about that, have a look at https://linuxize.com/post/how-to-configure-git-username-and-email/

KetchupBomb commented:
Ok, sorry for the noise, @Krzmbrzl. I think I've got everything addressed which you pointed out:

* Removed additional identified lines
* Fixed your attribution
* Fixed my git config username and email

Let me know if I missed anything else.

Krzmbrzl reviewed (approved):
For the Co-Authored-By thingy a single mail address would have been enough (I usually use the dev@... one for this) ^^
But it seems that it also doesn't hurt to have them all in there :sweat_smile:

Krzmbrzl commented:
Thank you for your contribution :+1:
